Makefile

```diff
@@ -42,6 +42,10 @@ watch: clean
 	make clean-lib
 	BABEL_ENV=development ./node_modules/.bin/gulp watch
```

Comment:
Do we want to include this in the normal `watch`?

Reply:
Does make support running two commands at the same time?

Reply:
Good point, that will probably be too complicated. LGTM then.